.github/workflows/pr-stable-build.yml

```diff
@@ -0,0 +1,33 @@
+# allowing test inside PRs
+
+name: "PR build release"
+on:
+    push:
+        branches:
+          - '**'        # matches every branch
+          - '!master'   # excludes master
+
+env:
+    CI: false
+jobs:
+    publish-tauri:
+        strategy:
+            fail-fast: false
+            matrix:
+                platform: [macos-latest, ubuntu-20.04, windows-latest]
+        runs-on: ${{ matrix.platform }}
+        steps:
+            - uses: actions/checkout@v3
+            - name: setup node
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 16
+            - name: install Rust stable
+              uses: dtolnay/rust-toolchain@stable
+            - name: install dependencies (ubuntu only)
+              if: matrix.platform == 'ubuntu-20.04'
+              run: |
+                  sudo apt-get update
+                  sudo apt-get install -y libgtk-3-dev libwebkit2gtk-4.0-dev libappindicator3-dev librsvg2-dev patchelf
+            - name: install app dependencies and build it
+              run: yarn && yarn build
```

Comment:
This doesn't actually build the whole application. It just builds the frontend using `react-scripts`. We'd need `yarn tauri:build` instead if we wanna build the whole application.

```suggestion
              run: yarn && yarn tauri:build
```

Reply:
And not it's meant to build the frontend then 
`- uses: tauri-apps/tauri-action@v0` does build tauri app